### ISSUE TITLE
jQuery 3.1 released but jquery-rails still not support it

### ISSUE THREAD

**Issue by Sega100500** [rails/jquery-rails#225]
jQuery 3.1 released but jquery-rails still not support it

When gem will be update to use jQuery 3 ?

**Comment by SandHilt** [comment]
+1

**Comment by andyklimczak** [comment]
Since it's merged but not yet released, you can specify to build from master instead of release (for now):
`gem 'jquery-rails', git: 'https://github.com/rails/jquery-rails.git'`

**Comment by Sega100500** [comment]
@andyklimczak 
When it will be released?
I prefer use official release

**Comment by SandHilt** [comment]
@andyklimczak or
```ruby
gem 'jquery-rails', gitHub: 'rails/jquery-rails'
```
source: [Bundler](http://bundler.io/git.html)

**Comment by SandHilt** [comment]
Tag is only name to ref in branch.
You can refer to branch directly in your Gemfile if you wish...
```ruby
gem 'jquery-rails', gitHub: 'rails/jquery-rails', ref: '7127a3d' # Support to Jquery 3.x
```
or wait for tests and get with gem.

**Comment by andyklimczak** [comment]
Actually, jquery3 itself isn't committed into vendor/, so technically this does not support jquery3 yet.

**Comment by rafaelfranca** [comment]
Released in 4.2.1

**Comment by Sega100500** [comment]
MANY, MANY THANKS!
All works!
